sdk/containerregistry/service.projects

```diff
@@ -7,13 +7,18 @@
   the set of projects to match what type of SDK we are interested in building.
 
   SDKType values
-   - empty or "client" will build the new track 2 client/data libraries
+   - "all" will build all projects
+   - "client" will build the new track 2 client/data libraries
    - "data" will build the track 1 data plane libraries
    - "mgmt" will build the track 1 mgmt libraries
 -->
 
 <Project>
-  <ItemGroup Condition="'$(SDKType)' == '' or '$(SDKType)' == 'client'">
+  <ItemGroup Condition="'$(SDKType)' == 'all'">
+    <ProjectsToInclude Include="$(MSBuildThisFileDirectory)**\*.csproj" />
```

Comment:
Do we really want it to be all or just track 2 libraries? Perhaps just add an or condition on the track 2 blocks below.

Reply:
Let's start with all and see if there are any problems with that.

Reply:
That's what effectively happens to other directories without `service.projects` so we keep things consistent.

Reply:
If we really want this to be all we should probably not duplication the include here but instead condition away exclude below. Keep in mine we have already included all the necessary project this logic simply undoes some of those includes but if you want all then you just need to undo that removal step.

Reply:
Moved even more logic into the service.proj